keel/api/v1/document/views.py

```diff
@@ -32,14 +36,24 @@ def generate(self, request, format = 'json', **kwargs):
         try:
             doc = Documents.objects.get(doc_pk = doc_id)
```

Comment:
you have fetched the user on line 28....but unless you filter the doc using that user...???